tsconfig.json

```diff
@@ -21,5 +21,5 @@
     "target": "es5"
   },
   "exclude": ["node_modules"],
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx"]
+  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", ".eslintrc.js"]
```

Comment:
I'm pretty sure it can't based off [the docs0(https://eslint.org/docs/user-guide/configuring/configuration-files#configuration-file-formats) and my testing with using a `.ts` vs `.js` extension.